hazelcast/proxy/queue.py

```diff
@@ -84,6 +124,17 @@ def contains_all(self, items):
         return self._encode_invoke(queue_contains_all_codec, data_list=data_items)
 
     def drain_to(self, list, max_size=-1):
+        """
+        Transfers all available items to the given collection and removes these items from this queue. If a max_size is
+        specified, it transfers at most the given number of items. In case of a failure, an item can be exist in both
+        collections or none of them.
+
+        This operation may be more efficient than polling elements repeatedly and putting into collection.
+
+        :param list: (List), the collection where the items in this queue will be transferred.
+        :param max_size: (int), the maximum number items to transfer (optional).
+        :return: (int), number of transferred items.
```

Comment:
..., maximum number of items to.....

(Let's put a convenience for "param" definitions: After the colon, let's not use the article "the", and let's start with small letters.